mhs-reference-implementation/mhs/sender/tests/test_sender.py

```diff
@@ -0,0 +1,38 @@
+from unittest import TestCase
+from unittest.mock import Mock, sentinel
+
+from mhs.sender.sender import Sender
+
+WRAPPER_REQUIRED = "ebxml_wrapper_required"
```

Comment:
is this constant here required? can we not just import it from the Sender module

Reply:
It was really just because it's used in a couple of locations, so I wanted to make it easier to update in future.

Reply:
but you have the same constant and value in the `sender` file too? Would it make sense to reference that constant here too.